apps/public/public.go

```diff
@@ -368,9 +368,8 @@ type AcquireByUsernamePasswordOption interface {
 	acquireByUsernamePasswordOption()
 }
 
+// Deprecated: This API will be removed in a future release. Use a more secure flow instead. Follow this migration guide: https://aka.ms/msal-ropc-migration
```

Comment:
This comment should be a separate paragraph to render correctly in docs (see [Go documentation](https://go.dev/doc/comment#deprecations))